docs/src/Stack.doc.js

```diff
@@ -0,0 +1,126 @@
+import PageHeader from './components/PageHeader.js';
+
+const cards = [];
+const card = c => cards.push(c);
+
+card(
+  <PageHeader
+    name="Stack"
+    description={`
+      Stack is a layout component with a very limited subset of the props available to Box.
+
+      Use this component for vertical flex layouts.
+    `}
+  />
+);
+
+card(
+  <PropTable
+    Component={Stack}
+    props={[
+      {
+        name: 'children',
+        type: 'React.Node',
+      },
+      {
+        name: 'alignContent',
+        type: `"start" | "end" | "center" | "between" | "around" | "evenly" | "stretch"`,
+        defaultValue: 'stretch',
+        description:
+          "Aligns a flex container's lines within when there is extra space in the horizontal axis, similar to how justify-content aligns individual items within the vertical axis.",
+      },
+      {
+        name: 'alignItems',
+        type: `"start" | "end" | "center" | "baseline" | "stretch"`,
+        defaultValue: 'start',
+        description:
+          'Defines the default behaviour for how flex items are laid out along the horizontal axis on the current line. Think of it as the justify-content version for the vertical axis (perpendicular to the horizontal axis).',
+        href: 'layout',
+      },
+      {
+        name: 'alignSelf',
+        type: `"auto" | "start" | "end" | "center" | "baseline" | "stretch"`,
+        defaultValue: 'stretch',
+        description:
+          'Allows the default alignment (or the one specified by align-items) to be overridden for individual flex items.',
+      },
+      { name: 'fit', type: 'boolean', defaultValue: false },
+      {
+        name: 'flex',
+        type: '"grow" | "shrink" | "none"',
+        defaultValue: 'shrink',
+        description: `Defines how a flex item will be sized. "grow", equivalent to "flex: 1 1 auto", will size the Stack relative to its parent, growing and shrinking based on available space. "shrink", equivalent to "flex: 0 1 auto" (the browser default), allows the Stack to shrink if compressed but not grow if given extra space. Finally, "none", equivalent to "flex: 0 0 auto", preserves the Stack's size based on child content regardless of its container's size.`,
+      },
+      {
+        name: 'gap',
+        type: '0 .. 12',
+        defaultValue: 0,
+        description:
+          'Defines symmetrical vertical padding to be applied to each child.',
+      },
+      {
+        name: 'justifyContent',
+        type: `"start" | "end" | "center" | "between" | "around" | "evenly"`,
+        defaultValue: 'center',
+        description:
+          'Defines the alignment along the vertical axis. It helps distribute extra free space left over when either all the flex items on a line are inflexible, or are flexible but have reached their maximum size. It also exerts some control over the alignment of items when they overflow the line.',
+        href: 'layout',
+      },
+      {
+        name: 'wrap',
```

Comment:
@dangerismycat does this make sense for `<Stack />`? Aren't the items always stacked on top of each other / wrapped?